python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -1373,18 +1380,22 @@ def end_watch_event_logs(self, run_id, cb):
 
     # asset storage
 
+    @public
```

Comment:
I think this should not be public... I think you can get the same effect by calling `get_asset_keys`